src/amber/dsl/router.cr

```diff
@@ -25,13 +25,16 @@ module Amber::DSL
       router.add(%route)
     end
 
+    macro namespace(scoped_namespace)
+      {{with DSL::Router.new(self, valve, scope + scoped_namespace) yield}}
+    end
+
     {% for verb in RESOURCES %}
       macro {{verb.id}}(*args)
         route {{verb}}, \{{*args}}
         {% if verb == :get %}
         route :head, \{{*args}}
         {% end %}
-        route {{verb}}, \{{*args}}
```

Comment:
hmm, why was this added twice? nice catch!

Reply:
Maybe a rebase or a merge conflict :smile: